spec/ms_clean_up_job_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'spec_helper'
+
+RSpec.describe 'MeiliSearch::Rails::MSCleanUpJob' do
+  include ActiveJob::TestHelper
+
+  subject(:job) { MeiliSearch::Rails::MSCleanUpJob }
+
+  subject(:record) do
+    Book.create name: "Moby Dick", author: "Herman Mellville",
+                premium: false, released: true
+  end
+
+  let(:record_entries) do
+    record.ms_entries(true).each { |h| h[:index_uid] += '_test' }
+  end
+
+  let(:indexes) do
+    %w[SecuredBook BookAuthor Book].map do |uid|
+      Book.index(safe_index_uid uid)
+    end
+  end
+
+  it 'removes record from all indexes' do
+    indexes.each(&:delete_all_documents)
+
+    record
```

Comment:
I need to say that I think just calling the creation of the AR record like this is weird to me, the line seems lost when I read the whole suite 😅

Reply:
A `let!` wouldn't do the trick?

Reply:
I attempted to refactor it, let me know if I should try again.